src/Helpers/AuditResolver.php

```diff
@@ -7,18 +7,18 @@
 
 class AuditResolver
 {
-    public static function findUserId(?Model $model): ?int
+    public static function findOwner(?Model $model): ?Model
     {
         if (empty($model)) {
```

Comment:
I think we should probably have a contract or interface for the model that we check here, but if we don't want that we should at least ensure the method exists before attempting to call it.

```suggestion
        if (empty($model) || !method_exists($model, 'trackableOwnerResolver')) {
```

Reply:
I liked this idea, so I added a `Trackable` contract in `Sourcetoad\Logger\Contracts` that defines `trackableOwnerResolver`. Instances of this contract are then checked in this static function's parameters.